hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -56,6 +56,29 @@ export const getTeamOrders = createAsyncThunk<
     }
 });
 
+export const updateOrder = createAsyncThunk<
+    OrderInTable,
+    number,
+    { state: RootState; rejectValue: RejectValue; dispatch: AppDispatch }
+>(`${orderReducerName}/updateOrder`, async (order_id, { rejectWithValue }) => {
+    try {
+        const data = {
+            status: "Cancelled",
+            request: "string",
+        };
+        const response = await patch<APIListResponse<Order>>(
```

Comment:
Check swagger again, the return type for this api is not an api list response of orders